tpp-backend/remove-ubuntu-user.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -euo pipefail
+
+# lock the ubuntu user
+usermod -L ubuntu
+
+# expire the ubuntu user
+chage -E0 ubuntu
+
+# remove any ssh config
+rm -rf ~ubuntu/.ssh
+
+# remove the host ssh keys, we will no longer be able to ssh in after this
+rm /etc/ssh/ssh_host_*_key*
```

Comment:
verifying assumptions - do you need to restart ssh daemon after this? does it get auto-restarted?

Reply:
It doesn't get restarted, so the comment is kinda misleading, as you can ssh in if it hasn't been restarted.

But the next thing after this is to stop the VM, so it's not a problem.

I think probably best to just update the comment.